src/FSAL/FSAL_CORTXFS/kvsfs_methods.h

```diff
@@ -10,18 +10,10 @@
 /******************************************************************************/
 /* Common helpers */
 
-static inline void cortxfs_cred_from_op_ctx(cfs_cred_t *out)
-{
-	assert(out);
-	assert(op_ctx);
-
-	out->uid = op_ctx->creds->caller_uid;
-	out->gid = op_ctx->creds->caller_gid;
-}
-
-#define CFS_CRED_INIT_FROM_OP {			\
-	.uid = op_ctx->creds->caller_uid,	\
-	.gid = op_ctx->creds->caller_gid,	\
+#define CFS_CRED_INIT_FROM_OP {					\
```

Comment:
Are we still using this? I see you have removed it from almost everywhere.

Reply:
Nope  .. I was under impression that I removed this. I will push new commit to remove this macro.

Reply:
fixed it.